### ISSUE TITLE
How to edit data in "SR_EV_UPDATE" without invoking the callback?

### ISSUE THREAD

**Issue by amitnagraj** [sysrepo/sysrepo#2970]
Hi Michal,

I'm using Sysrepo v2.0.47

When we set or delete an item inside "SR_EV_UPDATE" callback, the changes that are made in this callback will be part of "SR_EV_CHANGE" and "SR_EV_DONE" callback.

Is there any option or an API that we can use to avoid these changes in change and done callback, which will be like editing the datastore with having any subscription on it. But the same i want to do it with subscription.....

Thanks..

**Comment by michalvasko** [comment]
> Is there any option or an API that we can use to avoid these changes in change and done callback

No, there is not, the callbacks need to report the actual changes.
